src/random.rs

```diff
@@ -82,6 +82,12 @@ fn additional_data() -> Result<[u8; 32], RandomnessError> {
 pub fn random_bytes() -> Result<[u8; 32], RandomnessError> {
     let mut engine = sha256::HashEngine::default();
 
+    #[cfg(not(any(target_arch = "x86", target_arch = "x86_64")))]
+    if let Some(bytes) = hardware_randomness()? {
+        engine.input(&bytes);
+    }
```

Comment:
This is a noop.

https://github.com/wizardsardine/liana/blob/32dbb9490e044f15fb62a9f86815aaaa5b435d80/src/random.rs#L44-L47

Reply:
True, then I just leave it in? I can't see it used anywhere